test/embulk/input/test_mixpanel.rb

```diff
@@ -46,9 +47,13 @@ def setup_client
         end
       end
       def satisfy_task_ignore_start_time(expected_task)
-        satisfy{|input_task|
+        satisfy {|input_task|
           assert_not_nil(input_task[:job_start_time])
-          assert_equal(expected_task, input_task.merge(job_start_time: expected_task[:job_start_time]))
+          # This will also verify incremental column upper limit
+          assert_equal(input_task[:incremental_column_upper_limit], input_task[:job_start_time] - UPPER_LIMIT_DELAY * 1000)
+          expected_task[:job_start_time] = input_task[:job_start_time]
```

Comment:
Minor: might be a little bit easier to understand if we can keep previous version (`merge` and then `assert`), because it's easy to misunderstand `=` op to `assert_equal` when we are in `test` context.